.gitignore

```diff
@@ -113,3 +113,5 @@ venv.bak/
 *.session
 
 .DS_Store
+
+wusc_output
```

Comment:
How are these files generated? Local test data should be somewhere else on the file system to prevent accidental commits.

Reply:
I keep draft code schemes before uploading.